central/src/main/java/org/glowroot/central/repo/CentralConfigDao.java

```diff
@@ -94,14 +91,14 @@ void write(String key, Object config, String priorVersion) throws Exception {
             throw new OptimisticLockException();
         }
         String newValue = mapper.writeValueAsString(config);
-        boundStatement = updatePS.bind();
         int i = 0;
-        boundStatement.setString(i++, newValue);
-        boundStatement.setString(i++, key);
-        boundStatement.setString(i++, currValue);
-        results = session.update(boundStatement);
-        row = checkNotNull(results.one());
-        boolean applied = row.getBool("[applied]");
+        boundStatement = updatePS.bind()
+            .setString(i++, newValue)
+            .setString(i++, key)
+            .setString(i++, currValue);
+        AsyncResultSet asyncresults = session.update(boundStatement);
```

Comment:
We can't rename this variable to results, since result is already used in the same scope with a different type (ResultSet instead of AsyncResultSet).
This is here a "false" asynchronous call, since the ".get()" method is called in `session.update(boundStatement)`.

I'm pretty sure all these kind of blocking calls could be replaced with some efficient non-blocking calls with `CompletionStage`.

This will come later :)